src/components/DialogImage/DialogImage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Dialog, Avatar, AvatarProps } from "@radix-ui/themes";
+import { ImageIcon } from "@radix-ui/react-icons";
+
+export const DialogImage: React.FC<{
+  src: string;
+  size?: AvatarProps["size"];
+  fallback?: AvatarProps["fallback"];
+}> = ({ size = "8", fallback = <ImageIcon />, src }) => {
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <Avatar radius="small" src={src} size={size} fallback={fallback} />
```

Comment:
Could be `cursor: "pointer"`? So user will be able to see that images are clickable